docs/src/DeveloperDocumentation/documentation.md

```diff
@@ -123,7 +123,7 @@ existing entries. An easy way to do that is to add your new BibTeX entry,
 then run [bibtool](http://www.gerd-neugebauer.de/software/TeX/BibTool/en/)
 by invoking it as follows from the root directory of the Oscar.jl repository:
 
-    bibtool docs/oscar_references.bib -o docs/oscar_references.bib
+    bibtool -r .bibtoolsrc docs/oscar_references.bib -o docs/oscar_references.bib
```

Comment:
There is a similar comment at the top of `.bibtoolrsc`, can you please also change that?

Reply:
Good catch, done!